### ISSUE TITLE
[Bug] transparent & black edges state not retained on window resize

### ISSUE THREAD

**Issue by mggevaer** [bernhard-42/vscode-cadquery-viewer#8]
Resizing the VSCode window disables the "black edges" and "retained" option.

**Comment by bernhard-42** [comment]
Hmm, yes, seems to be a bug.
What do you mean by "retained"?

**Comment by mggevaer** [comment]
Just that both options reset to their default value, meaning disabled.

**Comment by bernhard-42** [comment]
I mean, I know what the "black edges" option is, I just can't map the word "retained" to, any option of the viewer

**Comment by mggevaer** [comment]
Ah, I just saw my error, Ignore the retained word, let me rephrase the issue.
If I turn on the option "black edges", and resize my VSCode window, the option is turned off again.
